polaris/ocean/tasks/isomip_plus/topo/remap.py

```diff
@@ -0,0 +1,140 @@
+        """
+        super().__init__(component=component, name=name, subdir=subdir)
+
+        self.set_shared_config(config, link='isomip_plus.cfg')
+
+        geom_filenames = dict(
+            ocean1='Ocean1_input_geom_v1.01.nc',
+            ocean2='Ocean2_input_geom_v1.01.nc',
+            ocean3='Ocean3_input_geom_v1.01.nc',
+            ocean4='Ocean4_input_geom_v1.01.nc')
+
+        if experiment not in geom_filenames:
+            raise ValueError(f'No input geometry for experiment {experiment}')
+
+        self.add_input_file(filename='topography.nc',
+                            target=geom_filenames[experiment],
+                            database='isomip_plus')
+        self.add_dependency(topo_map, name='topo_map')
+        self.add_output_file('topography_remapped.nc')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        self._preprocess()
+        self._remap()
+        self._rename()
+        self._renormalize()
+
+    def _preprocess(self):
+        ice_density = self.config.getfloat('isomip_plus', 'ice_density')
+
+        with (xr.open_dataset('topography.nc') as ds_in):
+            if 't' in ds_in.dims:
+                ds_in = ds_in.chunk({'t': 1})
+            ds_in['iceThickness'] = ds_in.upperSurface - ds_in.lowerSurface
+            ds_in.iceThickness.attrs['description'] = 'ice thickness'
+            ds_in.iceThickness.attrs['units'] = 'm'
+
+            gravity = constants['SHR_CONST_G']
+            ds_in['landIcePressure'] = (ice_density * gravity *
+                                        ds_in.iceThickness)
+            ds_in.iceThickness.attrs['description'] = \
+                'pressure at the ice base'
+            ds_in.iceThickness.attrs['units'] = 'Pa'
+
+            ds_in = ds_in.rename({'floatingMask': 'landIceFloatingFraction',
+                                  'groundedMask': 'landIceGroundedFraction',
+                                  'openOceanMask': 'openOceanFraction',
+                                  'lowerSurface': 'landIceDraft'})
+            ds_in['oceanFraction'] = \
+                (ds_in.bedrockTopography < 0.).astype(float)
+            ds_in['landIceFraction'] = (ds_in.landIceFloatingFraction +
+                                        ds_in.landIceGroundedFraction)
+
+            ds_in.to_netcdf('topography_processed.nc')
+
+    def _remap(self):
+        logger = self.logger
+        topo_map = self.dependencies['topo_map']
+
+        remapper = topo_map.get_remapper()
+
+        remapper.remap_file(inFileName='topography_processed.nc',
+                            outFileName='topography_ncremap.nc',
+                            logger=logger)
+
+    @staticmethod
+    def _rename():
+        with xr.open_dataset('topography_ncremap.nc') as ds_out:
+            drop = ['x', 'y', 'area', 'lat_vertices', 'lon_vertices']
+            rename = {'ncol': 'nCells'}
+            if 't' in ds_out.dims:
+                drop.append('t')
```

Comment:
Why drop the variable if you are going to rename it?

Reply:
I believe the `t` coordinate is being dropped and the `t` dimension is being renamed to `Time` but I don't honestly recall and I wish I'd made a comment here.  I'll add one indicating that that's what I *think* I'm doing there.

Reply:
Oh, that would make some sense.

Reply:
I added a clarifying comment.